_posts/2018-01-03-Continuous_delivery_with_episerver_dxcs.md

```diff
@@ -0,0 +1,459 @@
+---
+layout: post
+title: Continuous delivery with Episerver DXC Service
+author: Rinorragi
+date: 2018-01-03 10:00:00 +0200
+excerpt: How to do continuous delivery with Episerver Digital Experiance Cloud Service. 
+categories: 
+- Episerver
+tags: 
+- Episerver 
+- DOTNET 
+- DevOps
+- DevSecOps
+- DevSec
+- PowerShell
+- Jenkins
+- CI
+- continuous delivery
+- continuous deployment
+- continuous integration
+---
+I have put a lot of hours into building up a continuous delivery environment which I can be proud of. I already wrote [in my previous blog](http://dev.solita.fi/episerver/2017/11/21/Installing-Jenkins-with-PowerShell-DSC.html) about how to setup a Jenkins for this purpose. It was a ground work where we start building in this blog post. This blog post is about how to do continuous Delivery with Episerver Digital Experiance Cloud Service but nothing stops you to use this information in some other place.
+
+## Continuous delivery (CD) and continuous integration (CI)
+
+For sure there are better references about CI and CD than following will be. Actually if you have hard time in convincing your product owner to continuous delivery we have [a blog](http://dev.solita.fi/2016/09/06/CD-for-UX-and-PO.html) for that purpose as well. For me continuous integration means merging work in git repository hourly with coworkers working on the same things. In the other hand continuous delivery means pushing the work to customer as fast as we can. This is fine and simple. People work on the code and then you deployment it once you are confident with it. When you speed up the cycle of your deployments your customer will have new features faster and you they can steer the direction of your development in a truly agile way. Characteristics of CI and CD are below. 
```

Comment:
Actually, if you...
On the other hand
the work to the customer
as fast as possible (?)
This is fine and simple. on ehkä vähän outo lause?
Team works on the codebase and then deploy it once they are confident with it (ainakin deployment -> deploy)
cycle of your deployments, your customer will have new features faster, and can steer the direction of your development in a truly agile way